psutil/_pslinux.py

```diff
@@ -386,8 +386,9 @@ def virtual_memory():
     mems = {}
     with open_binary('%s/meminfo' % get_procfs_path()) as f:
         for line in f:
-            fields = line.split()
-            mems[fields[0]] = int(fields[1]) * 1024
+            name, _, value = line.partition(':')
+            value_num, _, units = value.partition(' ')
+            mems[name + ':'] = int(value_num) * 1024
```

Comment:
I think it makes more sense to avoid adding `':'` here and instead remove the colon from the subsequent strings, namely these 2:

```python
    try:
        total = mems[b'SwapTotal:']
        free = mems[b'SwapFree:']
    except KeyError:
```